### PR TITLE
Rename "Unknown Place" pin title to "Map Point"

### LINKED ISSUE #6041: Rename "Unknown Place" pin title
It doesn't sound good in my opinion. If we can't reverse geocode a particular location for whatever reason, then "Dropped pin" like in Google Maps (e.g. in Polish: just "Pinezka") seems to be suitable.

### PR DESCRIPTION
**Pull request by meenbeese**
Resolves #6041 by following the discussion there.

Please submit your translation suggestions here.

PS: I will regenerate the strings once the PR is finalized, as this is easier to deal with.

### CODE REVIEW

File: data/strings/strings.txt [hunk 1]
```diff
@@ -14848,44 +14848,44 @@
...
+    ko = 지도 지점
+    lt = Žemėlapio taškas
+    mr = नकाशा बिंदू
+    nb = Kartpunkt
+    nl = Kaart punt
+    pl = Punkt mapy
+    pt = Ponto do mapa
+    pt-BR = Ponto do mapa
+    ro = Punct Hartă
+    ru = Точка карты
```
Comment (pastk) [inline data/strings/strings.txt:14880]: Точка на карте

File: data/strings/strings.txt [hunk 2]
```diff
@@ -14848,44 +14848,44 @@
...
+    be = Пункт карты
+    bg = Точка на картата
+    ca = Punt del mapa
+    cs = Bod na mapě
+    da = Kortpunkt
+    de = Kartenpunkt
+    el = Σημείο χάρτη
+    es = Punto del mapa
+    et = Kaardipunkt
+    eu = Maparen Puntua
```
Comment (fitojb) [inline data/strings/strings.txt:14863]: Title Case is unneeded in Basque; this can be "Maparen puntua".

Comment (alexgabi) [reply data/strings/strings.txt:14863]: Certainly "Maparen puntua" is the correct way.

File: data/strings/strings.txt [hunk 3]
```diff
@@ -14848,44 +14848,44 @@
...
+    fr = Point de la carte
+    hu = Térképpont
+    id = Titik Peta
+    it = Punto della mappa
+    ja = マップポイント
+    ko = 지도 지점
+    lt = Žemėlapio taškas
+    mr = नकाशा बिंदू
+    nb = Kartpunkt
+    nl = Kaart punt
```
Comment (jimcarst) [inline data/strings/strings.txt:14875]:
```suggestion
    nl = Kaartpunt
```

**Review by jimcarst — APPROVED** [review]

**Review by matheusgomesms — APPROVED** [review]
PT and PT-BR look good

File: data/strings/strings.txt [hunk 4]
```diff
@@ -14848,44 +14848,44 @@
...
+    cs = Bod na mapě
+    da = Kortpunkt
+    de = Kartenpunkt
+    el = Σημείο χάρτη
+    es = Punto del mapa
+    et = Kaardipunkt
+    eu = Maparen Puntua
+    fa = نقطه نقشه
+    fi = Karttapiste
+    fr = Point de la carte
```
Comment (Wivik) [inline data/strings/strings.txt:14866]:
```suggestion
    fr = Point sur la carte
```

**Review by Wivik — COMMENTED** [review]
French suggestion done.

File: data/strings/strings.txt [hunk 5]
```diff
@@ -14848,44 +14848,44 @@
...
-    th = ไม่ทราบชื่อสถานที่
-    tr = Bilinmeyen Yer
-    uk = Невідоме місце
-    vi = Địa Điểm Chưa Biết
-    zh-Hans = 未知位置
-    zh-Hant = 未知的位置
+    en = Map Point
+    af = Kaartpunt
+    ar = نقطة الخريطة
+    be = Пункт карты
```
Comment (biodranik) [inline data/strings/strings.txt:14854]:
```suggestion
    be = Месца на мапе
```

File: data/strings/strings.txt [hunk 6]
```diff
@@ -14848,44 +14848,44 @@
...
+    pl = Punkt mapy
+    pt = Ponto do mapa
+    pt-BR = Ponto do mapa
+    ro = Punct Hartă
+    ru = Точка карты
+    sk = Bod na mape
+    sv = Kartpunkt
+    th = จุดแผนที่
+    tr = Harita Noktası
+    uk = Точка карти
```
Comment (biodranik) [inline data/strings/strings.txt:14885]:
```suggestion
    uk = Точка на карті
```

**Review by biodranik — COMMENTED** [review]
I'm still not 100% convinced that it would be better, need to test this change in the app.

**Comment by meenbeese** [comment]
Will merge the suggestions shortly, I had to reset my Linux VM.

**Comment by pastk** [comment]
Any progress here?

**Comment by meenbeese** [comment]
> Any progress here?

Yes, sorry. I was too busy with other stuff, I'll try to wrap this up soon.

**Review by pastk — APPROVED** [review]
LGTM!

Please add a strings regeneration commit! (`tools/unix/generate_localizations.sh`)

**Review by d4f5409d — APPROVED** [review]
Hu approved

**Comment by pastk** [comment]
@meenbeese do you plan to finish it? do you need help with it?

**Review by d4f5409d — APPROVED** [review]
🇭🇺✅

**Comment by meenbeese** [comment]
> @meenbeese do you plan to finish it? do you need help with it?

Yes, sorry for the delay as I have been busy for some time. I assume that only a regeneration commit is needed, right? In that case, I will push that soon.

**Comment by pastk** [comment]
Needs a rebase also.

**Comment by meenbeese** [comment]
@biodranik Ready to merge. I can't regenerate strings due to a script error.

**Comment by pastk** [comment]
Could you please post the error log? Let's solve the issue!

**Comment by Jean-BaptisteC** [comment]
Rebase your branch, we can regenerate strings after merging PR

File: data/strings/strings.txt [hunk 7]
```diff
@@ -28978,7 +28978,7 @@
     eu = Zure dohaintza eta laguntzarekin, Munduko mapa onenak sor ditzakegu!
     fa = با کمک های مالی و حمایت شما، ما می توانیم بهترین نقشه های جهان را ایجاد کنیم!
     fi = Lahjoituksellasi ja tuellasi voimme luoda maailman parhaat kartat!
-    fr = Avec vos dons et votre soutien, nous pouvons créer les meilleures cartes du monde !
+    fr = Avec vos dons et votre soutien, nous pouvons créer les meilleures cartes du monde !
```
Comment (pastk) [inline data/strings/strings.txt:28981]: Why is this change here? (and further)
It looks like there was a non-breakable space, which makes sense

**Comment by pastk** [comment]
> I'm still not 100% convinced that it would be better, need to test this change in the app.

Would you prefer to put it through a beta first?

**Comment by pastk** [comment]
Could you please rebase and add strings regenerate commit?

**Comment by pastk** [comment]
Please also change in https://github.com/organicmaps/organicmaps/blob/9b39fc327779f0bee43205a6a30d5835ac79b9b1/map/framework.cpp#L307

Are you OK to add a strings regeneration commit?

**Comment by pastk** [comment]
@meenbeese Do you want to finish it yourself or maybe I can takeover?

**Comment by meenbeese** [comment]
> @meenbeese Do you want to finish it yourself or maybe I can takeover?

Rebased and applied suggestions. I prefer to do the regeneration commit after merging.
